### ISSUE TITLE
Docs: what does this module do?

### ISSUE THREAD

**Issue by morenoh149** [Andr3wHur5t/react-native-keyboard-spacer#2]
Hey there. I'm unclear as what this spacer does exactly. Here

``` javascript
      <View style={[{backgroundColor: 'green', flex: 1}]}>
        <TextInput style={{backgroundColor: 'yellow', flex: 1}}/>
        <View style={{backgroundColor: 'red', left: 0, right: 0, height: 5}}/>
        <KeyboardSpacer/>
      </View>
```

I would understand if `KeyboardSpacer` were between the textinput and view then I would assume the view ends up at the bottom as the spacer grows to all the space available. But what actually renders in the example code?

**Comment by Andr3wHur5t** [comment]
The red view shows that the other views changed size, I will change the example to make this more clear.

![Example Gif](http://zippy.gfycat.com/UnrulyBasicDore.gif)

**Comment by Andr3wHur5t** [comment]
Fixed in 076fa9c.
